ChartsDemo-macOS/PlaygroundChart.playground/Pages/CombinedChart.xcplaygroundpage/Contents.swift

```diff
@@ -47,7 +47,7 @@ func generateLineData() -> LineChartData
     set.fillColor = #colorLiteral(red: 0.941176470588235, green: 0.933333333333333, blue: 0.274509803921569, alpha: 1.0)
     set.mode = .cubicBezier
     set.drawValuesEnabled = true
-    set.valueFont = NSUIFont.systemFont(ofSize: CGFloat(10.0))
+    set.valueFont = NSUIFont.systemFont(ofSize: 10.0)
```

Comment:
here you used NSUIFont

Reply:
I forgot to remove it

Reply:
@liuxuan30  sorry for my delay. I might review it tomorrow!

Reply:
hello there. any update?

Reply:
@liuxuan30 sorry. I thought I had updated here. I'm going to open the project now

Reply:
I'm just checking if I can merge this PR today or tomorrow. Feel no pressure if you have other tasks. just let me know and I can merge this later and release a new version this week.

Reply:
@liuxuan30 thanks! I had some issues at my previous work and totally forget to update this PR.